app/controllers/citizens/accounts_controller.rb

```diff
@@ -1,6 +1,5 @@
 module Citizens
-  class AccountsController < BaseController
-    include ApplicationFromSession
+  class AccountsController < CitizenBaseController
     before_action :authenticate_applicant!
```

Comment:
we can remove all these now that `before_action :authenticate_applicant!` is in `CitizenBaseController`

Reply:
this comment also applies to other controllers